frontend/components/events/create/DescriptionImageForm.tsx

```diff
@@ -71,29 +59,14 @@ export function DescriptionImageForm({
             {imagePreviewUrl !== "" && (
               <Image src={imagePreviewUrl} width={0} height={0} alt="imagePreview" className="h-72 w-fit p-4" />
             )}
-            <div className="flex flex-wrap mt-4">
-              {/* {previousImages.map((imageUrl, index) => (
-                <div
-                  key={index}
-                  className="cursor-pointer mr-4 mb-4"
-                  onClick={() => handleImageSelection(imageUrl)}
-                >
-                  <Image
-                    src={imageUrl}
-                    width={100}
-                    height={100}
-                    alt={`Previous Image ${index}`}
-                    className="rounded-md"
-                  />
-                </div>
-              ))} */}
-            </div>
+            <div className="flex flex-wrap mt-4"></div>
             <input
               className="rounded-md mt-4 ml-4"
               accept="image/*"
               type="file"
               onChange={(e) => {
-                if (e.target.files !== null) {
+                if (e.target.files !== null && validateImage(e.target.files[0])) {
```

Comment:
the `files[0]` leads me to think that there is the possibility of multiple files?
Although I know that most of the time it would only be uploading a single image file, I think it would still be more correct to have not hard code to 0 and instead loop through the files, validating each of them